modules/core/src/main/java/org/apache/ignite/internal/processors/platform/cluster/PlatformClusterGroup.java

```diff
@@ -356,6 +359,22 @@ public PlatformClusterGroup(PlatformContext platformCtx, ClusterGroupEx prj) {
                 return TRUE;
             }
 
+            case OP_ENABLE_STATISTICS: {
+                boolean enabled = reader.readBoolean();
+
+                int cnt = reader.readInt();
+
+                Collection<String> cacheNames = new ArrayList<>(cnt);
+
+                for (int i = 0; i < cnt; i++) {
+                    cacheNames.add(reader.readString());
+                }
+
+                platformCtx.kernalContext().cache().enableStatistics(cacheNames, enabled);
```

Comment:
Please use public API when possible: `platformCtx.kernalContext().grid().cluster().enableStatistics(cacheNames, enabled)`

There is a difference in exception handling and `GridKernalGateway.readLock` call.